Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -34,29 +38,45 @@ struct SwiftUINativeCheckoutDemo: View {
         NavigationView {
             ZStack {
                 VStack(spacing: 16) {
-                    Picker("Checkout ", selection: $checkoutTypeSelection) {
-                        ForEach(CheckoutType.allCases) {type in
-                            Text(type.rawValue)
-                        }
+                    VStack {
+                        Form {
+                            Picker("Checkout ", selection: $checkoutTypeSelection) {
+                                ForEach(CheckoutType.allCases) {type in
+                                    Text(type.rawValue)
+                                }
+                            }
```

Comment:
Woah - pretty cool you can make an entire picker view with just this much code 🤯